build/vega-lite-schema.json

```diff
@@ -4918,25 +5103,48 @@
           "minimum": 0,
           "type": "number"
         },
+        "labelFontWeight": {
+          "$ref": "#/definitions/FontWeight",
+          "description": "The font weight of legend label."
+        },
         "labelLimit": {
-          "description": "Maximum allowed pixel width of axis tick labels.",
+          "description": "Maximum allowed pixel width of axis tick labels.\n\n__Default value:__ `160`.",
           "type": "number"
         },
         "labelOffset": {
           "description": "The offset of the legend label.",
-          "minimum": 0,
           "type": "number"
         },
+        "labelOverlap": {
+          "anyOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "enum": [
+                "parity"
+              ],
+              "type": "string"
+            },
+            {
+              "enum": [
+                "greedy"
+              ],
+              "type": "string"
+            }
+          ],
+          "description": "The strategy to use for resolving overlap of labels in gradient legends. If `false`, no overlap reduction is attempted. If set to `true` (default) or `\"parity\"`, a strategy of removing every other label is used. If set to `\"greedy\"`, a linear scan of the labels is performed, removing any label that overlaps with the last visible label (this often works better for log-scaled axes).\n\n__Default value:__ `true`."
+        },
         "offset": {
-          "description": "The offset, in pixels, by which to displace the legend from the edge of the enclosing group or data rectangle.\n\n__Default value:__  `0`",
+          "description": "The offset in pixels by which to displace the legend from the data rectangle and axes.\n\n__Default value:__ `18`.",
           "type": "number"
         },
-        "orient": {
```

Comment:
orient is supported in Vega Legend config: https://vega.github.io/vega/docs/config/#legends

Reply:
Oh, this is annoying. Orient is supported in config and legend but it does not support signals.